src/fundus/scraping/pipeline.py

```diff
@@ -67,6 +218,13 @@ def crawl(
         else:
             extraction_filter = only_complete
 
+        if isinstance(delay, float):
+
+            def _id(n: float) -> Delay:
```

Comment:
Why is this necesarry?

Reply:
Static type checkers like `mypy` discourage assigning lambda expressions to variables.